aggregator/src/aggregator/http_handlers.rs

```diff
@@ -68,25 +68,39 @@ impl Handler for Error {
             Error::MissingTaskId => {
                 conn.with_problem_document(&ProblemDocument::new_dap(DapProblemType::MissingTaskId))
             }
-            Error::UnrecognizedAggregationJob(task_id, _) => conn.with_problem_document(
-                &ProblemDocument::new_dap(DapProblemType::UnrecognizedAggregationJob)
-                    .with_task_id(task_id),
-            ),
-            Error::DeletedCollectionJob(_) => conn.with_status(Status::NoContent),
-            Error::AbandonedCollectionJob(collection_job_id) => conn.with_problem_document(
-                &ProblemDocument::new(
-                    "https://docs.divviup.org/references/janus-errors#collection-job-abandoned",
-                    "The collection job has been abandoned.",
-                    Status::InternalServerError,
-                )
-                .with_detail(concat!(
+            Error::UnrecognizedAggregationJob(task_id, aggregation_job_id) => conn
+                .with_problem_document(
+                    &ProblemDocument::new_dap(DapProblemType::UnrecognizedAggregationJob)
+                        .with_task_id(task_id)
+                        .with_aggregation_job_id(aggregation_job_id),
+                ),
+            Error::DeletedAggregationJob(task_id, aggregation_job_id) => conn
+                .with_problem_document(
+                    &ProblemDocument::new(
+                        "https://docs.divviup.org/references/janus-errors#aggregation-job-deleted",
```

Comment:
I'll make a change to `divviup/public-docs` so that this link resolves to something.

Reply:
https://github.com/divviup/public-docs/pull/49